### PR TITLE
Added the pull request template

### PR DESCRIPTION
**Pull request by donlingliang**
# Description
Adding a proposed pull request template for other contributors so that they just need to fill out the template before creating a pull request in this repo.

Feel free to check out the rich diff and comment on this proposal .

### CODE REVIEW

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Proposed changes
+
+Describe your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a featured tradingview script, be sure to link to that theory or video url from lonecapital.
```
Comment (Coding-Fun) [inline docs/pull_request_template.md:3]: Could you clarify on `resolves a featured tradingview script`?

Comment (Coding-Fun) [inline docs/pull_request_template.md:3]: Also nit: LoneCapital

File: docs/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Types of changes
+
+What types of changes does your code introduce to BankrollHunter?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New scripts (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation Update (if none of the other choices apply)
```
Comment (Coding-Fun) [inline docs/pull_request_template.md:13]: nit: lower case `update` to be consistent with the other lines?

File: docs/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+## Proposed changes
+
+Describe your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a featured tradingview script, be sure to link to that theory or video url from lonecapital.
+
+## Types of changes
```
Comment (Coding-Fun) [inline docs/pull_request_template.md:5]: What about new features to existing scripts?

File: docs/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation Update (if none of the other choices apply)
+
+## Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
+- [ ] I have tested the scripts with TradingView
+- [ ] I have added necessary documentation (if appropriate)
+- [ ] Any dependent changes have been merged and published in downstream modules
```
Comment (Coding-Fun) [inline docs/pull_request_template.md:21]: Q: What downstream modules we're going to have?

File: docs/pull_request_template.md (hunk 1 above)
Comment (lisangzhou) [reply docs/pull_request_template.md:3]: maybe Dong meant

"If the changes fix an existing bug or are derived from an existing tradingview script, be sure to reference that tradingview script or any theory from lonecapital"

File: docs/pull_request_template.md (hunk 2 above)
Comment (lisangzhou) [reply docs/pull_request_template.md:13]: +1

File: docs/pull_request_template.md (hunk 4 above)
Comment (lisangzhou) [reply docs/pull_request_template.md:21]: @Coding-Fun I think we should have a chat with lonecapital for additional ideas and things that we could work on. 

Personally, I don't think additional indicators are particularly useful. 

The things we could work on are:
1. gap highlighting. When ever there is a price gap, it's usually filled within days. Thus the gap can be used as a target for upward/downward price movement. We should highlight these gaps for improved visibility.
2. high volume highlighting. High volume indicates a large number of transactions. The prices around these high volume might indicate whether the trend would continue or pause.

Comment (lisangzhou) [reply docs/pull_request_template.md:21]: Once we have a feature roadmap, we can determine the technical questions. like what additional modules / tools we want.

File: docs/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+Describe your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a featured tradingview script, be sure to link to that theory or video url from lonecapital.
+
+## Types of changes
+
+What types of changes does your code introduce to BankrollHunter?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New scripts (non-breaking change which adds functionality)
```
Comment (lisangzhou) [inline docs/pull_request_template.md:11]: New script 

A PR should only contain one new script. If the person want to submit more than one script, each script should be in a PR.

File: docs/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```
Comment (lisangzhou) [inline docs/pull_request_template.md:12]: I am actually not sure why we'd have breaking change. These scripts aren't APIs so we shouldn't worry about backward compatibility.

File: docs/pull_request_template.md (hunk 3 above)
Comment (lisangzhou) [reply docs/pull_request_template.md:5]: mm great question. I think in that case, we should discuss in the slack channel first.

**Review by lisangzhou — COMMENTED** [review]
Some quick comments and questions, but mostly LGTM. 

I'd also consider moving the template to `.github/pull_request_template.md` so it's in a hidden folder, but that's just personal preference. I tend to keep actual docs in `/docs`. 

Excellent work, sir!

**Comment by donlingliang** [comment]
i feel like we just need a `docs` folder instead of a hidden one.
other than that, lets discuss in channel. thanks.

**Review by u2cc — COMMENTED** [review]
All is good. May I know if all reviewers need to approve before PR can be merged, please? If so I will provide explicit approval to this PR in GitHub. Thanks.

**Comment by donlingliang** [comment]
> All is good. May I know if all reviewers need to approve before PR can be merged, please? If so I will provide explicit approval to this PR in GitHub. Thanks.

i usually do one assignee, but multiple reviewers. i think we need to talk about this template in channel though.

**Review by kentio — APPROVED** [review]
very useful

**Pull request merged**